src/monero_prehash.c

```diff
@@ -56,6 +56,9 @@ int monero_apdu_mlsag_prehash_init() {
         // fee str
         monero_vamount2str(G_monero_vstate.io_buffer + G_monero_vstate.io_offset,
                            G_monero_vstate.ux_amount, 15);
+
```

Comment:
Just a question: why is this needed in the context of Flex Porting??

Reply:
this change has been mentioned in the PR header. 
It's an UI improvement asked by @vforgeoux-ledger